src/Classes/ParcelPackage.ps1

```diff
@@ -2,7 +2,7 @@ class ParcelPackage
 {
     [string] $Name
     [string] $ProviderName
-    [string] $Source
+    [array] $Source
```

Comment:
Make this a `[string[]]` rather than an `any object array`. I'd also make sure this doesn't break any of the other providers; in theory it shouldn't, as powershell normally treats a single item array as just a normal item 🤔

Reply:
Ahh i was hoping it would treat any single entry array as a string here and that wouldn't break anything. Done anyway